Dockerfile.web

```diff
@@ -0,0 +1,27 @@
+FROM node:10.16.3 as builder
+
+WORKDIR /app
+ADD . .
+
+ARG REACT_APP_INFURA_KEY
+RUN yarn install
+RUN yarn workspace @chainlink/heartbeat run build
+
+FROM node:10.16.3-alpine
+
+WORKDIR /app
+
+COPY --from=builder /app/package.json package.json
+COPY --from=builder /app/yarn.lock yarn.lock
+COPY --from=builder /app/node_modules node_modules
+COPY --from=builder /app/feeds_ui/node_modules feeds_ui/node_modules
+COPY --from=builder /app/feeds_ui/package.json feeds_ui/package.json
+COPY --from=builder /app/feeds_ui/server.js feeds_ui/server.js
+COPY --from=builder /app/feeds_ui/src feeds_ui/src
+COPY --from=builder /app/feeds_ui/src/feeds.json feeds_ui/build/feeds.json
+COPY --from=builder /app/feeds_ui/src/nodes.json feeds_ui/build/nodes.json
+COPY --from=builder /app/feeds_ui/public feeds_ui/public
+COPY --from=builder /app/feeds_ui/build feeds_ui/build
+
+ENV NODE_ENV production
+ENTRYPOINT [ "yarn", "workspace", "@chainlink/heartbeat", "run", "start" ]
```

Comment:
What do you think about having different naming for `feeds_ui` being called `heartbeat`? I think it could be confusing at a glance for people not as familiar with the codebase. Should we rename one or another to keep it consistent?

Reply:
I agree that the naming here is pretty awkward. What do you think this app/dir should be called @hellobart?

Reply:
Agree, We should not name it `heartbeat`, as not all of the data reference are based on heartbeat (like prepaid aggregator or pre coordinators). I would stick with `feeds`.

Reply:
@HenryNguyen5 @hellobart updated in https://github.com/smartcontractkit/chainlink/pull/2208 `feeds_ui/` dir -> `feeds/`

Also updated this PR so that it's running the new package name `@chainlink/heartbeat` -> `@chainlink/feeds`